### PR TITLE
1485512 improve import all button test coverage

### PR DESCRIPTION
**Pull request by C2H6Ethan**
[//]: # 'This is the default template for Merge Requests'

## Test description

**As** a User  
**I want to** use the plugin bug free  
**So that** I can work more efficiently 

### Important
I placed the test inside the queryResults.cy.tsx since the calculation I'm testing, is being done inside QueryResults.tsx and then that result is passed as a property to ImportActions.tsx.
Let me know if this is okay.

### Changes

| File         | Changes                     |
| ------------ | --------------------------- |
| `editorextensions/codebeamer-custom-ui/modal/src/pages/import/components/queryResults/queryResults.cy.tsx` |Added a test to make sure the calculation of unimportedItems is correct|
|`editorextensions/codebeamer-custom-ui/modal/src/pages/import/components/importActions/importActions.cy.tsx`|Added the trackerId property to item arrays since I added that property to the CardBlockToItemMapping interface in the [fix Import all button task](https://retina.roche.com/cb/issue/1457451)|
|`editorextensions/codebeamer-custom-ui/modal/src/pages/import/components/updater/updater.cy.tsx`|Same as the change above|

## Testing approach

[//]: # 'Describe how the functionality is / can be tested'
[//]: # 'If there are unit tests, describe them'

**Scenario**: Have items from different trackers on the lucidspark board  
**Given** I open the import modal
**When** I select a tracker
**Then** I see the amount of items from the tracker that have not been imported yet displayed on the Import All button

**Scenario**: Have duplicate items on the lucidspark board  
**Given** I open the import modal
**When** I select a tracker
**Then** I see the amount of items from the tracker that have not been imported yet displayed on the Import All button

### Assignee

-   [x] Adequate Milestone set :triangular_flag_on_post:  
-   [x] Adequate labels set :label:  
-   [ ] Affected documentation updated :books:
    - [ ] Changelog :bookmark_tabs:
    - [ ] Wiki :blue_book:

### Reviewer

-   [ ] Testing
    -   [ ] Coverage :ok_hand:
    -   [ ] Quality :ok_hand:
    -   [ ] Evergreen :100:
-   [ ] Code Quality
    -   [ ] :zero: new lint errors :poop:  
    -   [ ] Maintainable :tools:
    -   [ ] Good patterns :third_place:

### CODE REVIEW

**Comment by urecha** [comment]
Yeah, test location works for me.

File: editorextensions/codebeamer-custom-ui/modal/src/pages/import/components/queryResults/queryResults.cy.tsx [hunk 1]
```diff
@@ -131,6 +131,92 @@ describe('<QueryResults>', () => {
...
+		const store = getStore();
+		store.dispatch(setTrackerId('4877085'));
+
+		cy.intercept('POST', `**/api/v3/items/query`, {
+			fixture: 'query_multi-page.json',
+		}).as('itemQuery');
+
+		cy.mountWithStore(<QueryResults />, { reduxStore: store });
+
+		cy.getBySel('importAll').should('have.text', 'Import all (11)');
```
Comment (urecha) [inline editorextensions/codebeamer-custom-ui/modal/src/pages/import/components/queryResults/queryResults.cy.tsx]: The assertion is a little very magic with the `11` coming out of nowhere. Will make it hard for anyone else to maintain / update. We'd do better if this calculation happened in an understandable way, for example by importing the fixture and taking the value from there in-code, or at the very least assigning the value to a clearly named constant beforehand.

Comment (C2H6Ethan) [reply editorextensions/codebeamer-custom-ui/modal/src/pages/import/components/queryResults/queryResults.cy.tsx]: calculating a expectedCount by using the fixtures

**Review by urecha — APPROVED** [review]
That _is_ to the point.

**Pull request merged**